packages/@n8n/nodes-langchain/nodes/agents/Agent/agents/utils.ts

```diff
@@ -17,3 +18,24 @@ export async function extractParsedOutput(
 	// with fallback to the original output if it's not present
 	return parsedOutput?.output ?? parsedOutput;
 }
+
+export async function checkForStructuredTools(
```

Comment:
This function is untested, would it be difficult to add a unit-test for it?

Reply:
It wouldn't, added a couple of unit tests for it